gettsim/social_insurance_contributions/ges_pflegev.py

```diff
@@ -23,8 +23,8 @@ def ges_pflegev_zusatz_kinderlos(
     -------
 
     """
-    # Todo: No hardcoded 22.
-    return ~hat_kinder & alter.gt(22)
+    out = soz_vers_beitr_params["ges_pflegev_zusatz_kinderlos"]["altersgrenze"]
+    return ~hat_kinder & alter.gt(out)
```

Comment:
Note from above, just as reminder: Rather use 23 and adjust the formula accordingly. Try to stay as close as possible to law.